docker-compose.yml

```diff
@@ -9,8 +9,101 @@ services:
       SA_PASSWORD: "Mtc-D3v.5ql_S3rv3r"
       ACCEPT_EULA: "Y"
       HOMEBREW_NO_ENV_FILTERING: 1
+  functions_base:
+    build: ./deploy/docker/functions-base
+    command: echo "mtc-functions-base built"
+  migration:
+    build:
+      context: ./admin
+      dockerfile: dev.Dockerfile
+    container_name: mtc_admin_dev_migration
+    volumes:
+      - "./:/mtc"
+    environment:
+      SQL_SERVER: 'sqldb'
+      SQL_PORT: '1433'
+    depends_on:
+      - sqldb
+    command: /bin/bash /mtc/admin/run-dev-migrations.sh
   redis:
     image: "redis:4.0-alpine"
     container_name: mtc_redis
     ports:
       - "6379:6379"
+    depends_on:
+      - sqldb
+      - migration
+  functions_app:
+    build: ./functions-app
+    container_name: "mtc_functions_app"
+    ports:
+      - 7072:7072
+    volumes:
+      - "./:/mtc"
+    environment:
+      SQL_SERVER: 'sqldb'
+      REDIS_HOST: 'redis'
+    depends_on:
+      - sqldb
+      - migration
+      - redis
+    command: ["./wait-for-db.sh", "sqldb", "yarn", "start"]
+  func_consumption:
+    build: ./func-consumption
```

Comment:
We should probably change the Dockerfiles in `func-consumption` and `functions-app` to be `dev.Dockerfile` as these are not prod assets, and the name change will make that clearer.